### PR TITLE
#1142 added html/xml view class

### PR DESCRIPTION
**Pull request by krzyk**
#1142 
* Added `View` class with XML and HTML view of given zerocracy page
* Refactored test classes to use the new View class

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1371](https://github.com/zerocracy/farm/issues/1371) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1371](https://github.com/zerocracy/farm/issues/1371) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1881549436); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/zerocracy/tk/View.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                new RqWithUser(
+                    this.farm,
+                    new RqFake(
+                        new ListOf<>(
+                            String.format("GET %s", this.url),
+                            "Host: www.example.com",
+                            "Accept: application/xml",
+                            agent
+                        ),
+                        ""
```
[the remark this answers is not in the document]
Comment (krzyk) [reply src/test/java/com/zerocracy/tk/View.java]: @amihaiemil a request body, `RqFake` doesn't have any constructor that takes only headers

**Comment by amihaiemil** [comment]
@krzyk Which test would be duplicated exactly? :D 

I think the class should be tested, from principle. It's a tool used in other tests, which may not behave correctly and that doesn't necessarily mean that the tests using it would fail; it may also mean that they pass when in fact they should fail (false positives).

**Comment by krzyk** [comment]
@amihaiemil todo added

**Comment by amihaiemil** [comment]
@krzyk thanks

**Comment by amihaiemil** [comment]
@rultor good to merge

**Comment by krzyk** [comment]
@g4s8 can we merge?

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1371](https://github.com/zerocracy/farm/issues/1371) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@amihaiemil According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by amihaiemil** [comment]
@ypshenychka I confirm :D

**Comment by ypshenychka** [comment]
@amihaiemil Thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Pull request merged**